### PR TITLE
PodEvictionTimeout is a variable which needs to be passed to  for master nodes

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1770891

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio PTAL

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -798,6 +798,23 @@ networkConfig:
...
 
+.Pod and Node Configuration Parameters
+[cols="3a,6a",options="header"]
+|===
+
+| Parameter Name | Description
+
+|`*PodEvictionTimeout*`
+|Controls grace period for deleting pods on failed nodes. It takes valid time
+duration string. If empty, you get the default pod eviction timeout.
```
Comment (sunilcio) [inline install_config/master_node_configuration.adoc:809]: @mburke5678 hi, I we need to move `PodEvictionTimeout` to master configuration https://docs.openshift.com/container-platform/3.11/install_config/master_node_configuration.html#master-node-config-kubernetes-master-config

Comment (sunilcio) [reply install_config/master_node_configuration.adoc:809]: also include statement that the default value is 5 minutes. Default: 5m0s

Comment (mburke5678) [reply install_config/master_node_configuration.adoc]: @sunilcio Does the `ProxyClientInfo` parameter also need to move to this table? The example master config in the BZ shows the variable.
https://bugzilla.redhat.com/show_bug.cgi?id=1770891#c1

Comment (sunilcio) [reply install_config/master_node_configuration.adoc]: @mburke5678 yes, I see it in master config. @rphillips hi, could you please confirm, thanks!

**Pull request merged**